Marlin/planner.h

```diff
@@ -166,6 +166,11 @@ class Planner {
      * Nominal speed of previous path line segment
      */
     static float previous_nominal_speed;
+	
+	/**
+	 * Limit where 64bit math is necessary for acceleration calculation
+	 */
+	static unsigned long cutoff_32bit;
```

Comment:
That said, we're preferring `uint32_t` over `unsigned long` lately. It's a matter of style, but also it's meant to be more specific.